brave/brave-core
# Add "channel" and "version" property to Android webcompat reports

Resolves https://github.com/brave/brave-browser/issues/36478, https://github.com/brave/brave-browser/issues/40678



## Submitter Checklist:

  * `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests` [wiki](https://github.com/brave/brave-browser/wiki/Tests)
  * `npm run presubmit` [wiki](https://github.com/brave/brave-browser/wiki/Presubmit-checks), `npm run gn_check`, `npm run tslint`

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:

Make sure that Webcompat Report contains the `version` and `channel` information. 
Steps:
1. Open Brave Shields pop up dialog
2. Switch off Brave Shields if It is ON
3. Click "Report broken site", then click "Submit"
4. Go to the webcompat reporter dashboard and see the `version` and `channel` information.
